src/linear_algebra/gram_schmidt_ortho.lean

```diff
@@ -0,0 +1,259 @@
+/-
+Copyright (c) 2022 Jiale Miao. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jiale Miao, Kevin Buzzard
+-/
+
+import tactic.basic
+import algebra.big_operators.basic
+import analysis.inner_product_space.basic
+import analysis.inner_product_space.projection
+import analysis.normed_space.is_R_or_C
```

Comment:
```suggestion
import analysis.inner_product_space.projection
```
This is the only one that's actually needed, everything else is imported transitively. (You almost never need to import `tactic.basic` explicitly.)

Reply:
oh yes, I found other ones unnecessary here. Thank you! I have fixed it